src/core/pubkey.zig

```diff
@@ -54,6 +57,13 @@ pub const Pubkey = extern struct {
     ) !void {
         return base58.format(self.data, writer);
     }
+
+    pub fn jsonParse(_: Allocator, source: anytype, _: ParseOptions) !Pubkey {
```

Comment:
Any reason to have the `jsonParse` function take an `Allocator` and `ParseOptions` even though these two are not used within the function?

Reply:
Yes. This function exists so that std.json can call it in a generic context. It always passes the same parameters to any instance of jsonParse.